package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "parser-ts",
-  "version": "0.6.7",
+  "version": "0.6.8",
```

Comment:
You can leave the `parser-ts` version number as is - it will be updated on release, so no need to bump here.